Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/SyncIntTests.swift

```diff
@@ -707,10 +711,10 @@ class SyncIntTests: BaseStitchIntTestCocoaTouch {
 
         // configure Sync to resolve a custom document on conflict.
         // sync on the id, and do a sync pass
+        try coll.sync(ids: [doc1Id])
```

Comment:
[q] why does this have to happen before configuration? wouldn't it be weird if users have to sync their document ids before they configure their synced collections?